### ISSUE TITLE
[SUGGESTION]: Add hotkeys to travel to next/previous town

### ISSUE THREAD

**Issue by YetiSpaghetti** [pokeclicker/pokeclicker#3907]
### Current Behavior

Currently there is no quick way to travel through towns other than clicking them on the map.

### Improved Behavior

Similar to how it's possible to move between routes using the `-` and `=` hotkeys, it would be awesome to be able to quickly travel through cities/towns in a similar manner.

When you are in a town, `=` should bring you to the "next" town, and `-` should bring you to the "previous" town.

The order of towns can probably be based on unlocking order, similar to the way route ordering is not based on route number but rather the order in which they were unlocked

### Reason for change

Minor QoL feature

Usually desired when I'm looking for something specific in a shop and I don't know off the top of my head which town it is in (and don't feel like looking up the wiki).

Matches functionality provided by the route navigation hotkeys,.

### Detailed Description

Not much more to add

**Comment by DraKay** [comment]
Is + not =

**Comment by YetiSpaghetti** [comment]
> Is + not =

true, I put `=` because the key to trigger it is `=/+` but I see the settings page says `+` so I'll update to that

**Comment by DraKay** [comment]
Oh, didn't know English qwerty shared the key

**Comment by RegisCoaxans** [comment]
Both + and = work as + because they share the same key in most keyboards.
